src/tests/ftest/osa/osa_offline_reintegration.py

```diff
@@ -72,7 +74,8 @@ def run_ior_thread(self, action, oclass, api, test):
         process.join()
 
 
-    def run_offline_reintegration_test(self, num_pool, data=False):
+    def run_offline_reintegration_test(self, num_pool, data=False,
+                                       server_boot=False):
```

Comment:
Can we please add the server_boot param description in doc string below.

Reply:
Done